src/tile/tile.ts

```diff
@@ -180,23 +180,23 @@ export const findSiblings = (
   const siblingsCoords = {
     topLeft: {
       x: tileCoords.x - 1,
-      y: tileCoords.y,
+      y: tileCoords.x <= 3 ? tileCoords.y : tileCoords.y - 1,
```

Comment:
It may be easier to understand with a constant.

`const MIDDLE_LINE_X_VALUE = 3;`

Reply:
Done